src/main/java/com/google/sps/servlets/HandlePhotoShoppingServlet.java

```diff
@@ -14,40 +14,36 @@
 
 package com.google.sps.servlets;
 
+import static javax.servlet.http.HttpServletResponse.SC_INTERNAL_SERVER_ERROR;
```

Comment:
I think the placement of this import was okay because it goes with the other javax.servlet imports, and also alphabetically.

Reply:
It was done automatically by the google-code-formatter.

Reply:
Alright then I think it's okay :)